components/form/layout/orientation.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Orientation
+page_title: Form Orientation
+description: Form for Blazor - Orientation.
+slug: form-orientation
+tags: telerik,blazor,form,edit,form,orientation
+published: True
+position: 0
+---
+
+# Form Orientation
+
+The Form component for Blazor allows you control its orientation through the `Orientation` parameter. It takes a member of the `Telerik.Blazor.Enums.FormOrientation` enum:
+
+* `Horizontal`
```

Comment:
Consider adding 1-2 sentences that describe the behavior so it is clear what it does

Reply:
Done